File: 2021/11/17/main.go
package main

func uniquePaths(m int, n int) int {
	dp := make([]int, n+1)
	for i := 0; i <= n; i++ {
		dp[i] = 1
	}

	for i := 2; i <= m; i++ {
		for j := 2; j <= n; j++ {
			dp[j] += dp[j-1]
		}
	}

	return dp[n]
}

File: 2021/11/17/main.py
class Solution:
    def uniquePaths(self, m: int, n: int) -> int:
        dp = [1] * (n+1)

        for i in range(2, m+1):
            for j in range(2, n+1):
                dp[j] += dp[j-1]

        return dp[n]